tsconfig.json

```diff
@@ -67,7 +66,7 @@
                 "ListControl": "modules/_commoncontrols_listcontrol_listcontrol_",
                 "DateControl": "modules/_commoncontrols_datecontrol_",
```

Comment:
does this also need an update?

Reply:
No, I had to update number control alone since it was moved under a numbercontrol/ dir with apl support files from top level numberControl.ts. Rest (ValueControl and DateControl) are still in top level under commonControls/